docs/helm-chart/manage-dags-files.rst

```diff
@@ -91,10 +96,27 @@ This option will use a Persistent Volume Claim with an access mode of ``ReadWrit
       # by setting the  dags.persistence.* and dags.gitSync.* values
       # Please refer to values.yaml for details
 
+When using ``apache-airflow>=2.0.0``, :ref:`DAG Serialization <apache-airflow:dag-serialization>` is enabled by default,
+hence Webserver does not need access to DAG files, so you can turn off ``git-sync`` for Webserver by setting
+``dags.gitSync.excludeWebserver`` to ``true``.
+This is also recommended when enabling DAG Serialization for ``apache-airflow>=1.10.7,<2``.
```

Comment:
For Airflow 1.10.9 and earlier, DAG files is still needed for DAG code ttab. For Airflow 1.10.10 and newer (including 2.0.0), it is needed to enable`store_dag_code` to `True` (default value: `False`). DAG Serialization is not enough. 
http://airflow.apache.org/docs/apache-airflow/2.0.1/configurations-ref.html#store-dag-code

Reply:
We had removed the default and have a fallback of `True` for `store_dag_code`

https://github.com/apache/airflow/blob/5da831910c358ecbd7a5c33ee31fe0d909508bea/airflow/settings.py#L460-L462

https://github.com/apache/airflow/blob/5da831910c358ecbd7a5c33ee31fe0d909508bea/airflow/config_templates/default_airflow.cfg#L205-L209

Same for 1.10.11:

https://github.com/apache/airflow/blob/1.10.11/airflow/config_templates/default_airflow.cfg#L237-L242

where the value is same as `store_serialized_dags`:

https://github.com/apache/airflow/blob/1.10.11/airflow/settings.py#L431-L434

Reply:
I updated the description in https://github.com/apache/airflow/pull/15314/commits/fb10b097e4537e46ecdc46c69bba199162324d71